README.md

```diff
@@ -4,3 +4,11 @@
 
 We're using use-query-params to store part of the app's state in the url query
 parameters. This allows users to bookmark and share specific states of the app.
+
+## Model overview
+
+The category model can be fairly confusing, so we've created a few diagrams to help explain and get an overview of how DHIS2 models are used in the app.
+
+[Models used in form](./docs/category-combo-diagram.png)
```

Comment:
shouldn't the links in github markup just be `(docs/category-combo-diagram.png)`?